src/index.ts

```diff
@@ -4,13 +4,14 @@ import { Options, Result } from "./types";
 
 export async function githubAppJwt({
   id,
-  privateKey
+  privateKey,
+  timeDifference = 0,
 }: Options): Promise<Result> {
   // When creating a JSON Web Token, it sets the "issued at time" (iat) to 30s
   // in the past as we have seen people running situations where the GitHub API
   // claimed the iat would be in future. It turned out the clocks on the
   // different machine were not in sync.
-  const now = Math.floor(Date.now() / 1000) - 30;
+  const now = Math.floor(Date.now() / 1000) - 30 + timeDifference;
```

Comment:
I wonder if instead of adding a `timeDifference` option, we should add a `now` option, which would default to `Math.floor(Date.now() / 1000)`. What do you think?

Reply:
I like that, I wasn't sure how much control to give to the user as far as just giving the time difference or giving everything (such as the iat _and_ exp). I think specifying `now` is a good middle ground where this library can build the `iat` and `exp` from the now value.

I can make that change and update the tests!